fastlane/metadata/android/en-US/full_description.txt

```diff
@@ -1 +1 @@
-<i>LibreTube</i> is an alternative frontend for YouTube. It uses the <a href='https://github.com/TeamPiped/Piped' target='_blank' rel='nofollow'>Piped</a> API to load data and play videos and thus needs no YT account and no Google Services on your device.
+<i>LibreTube</i> is an alternative frontend for YouTube. It optionally allows you to use <a href='https://github.com/TeamPiped/Piped' target='_blank' rel='nofollow'>Piped's</a> API to load data and play videos for better privacy as well as syncing your subscriptions and playlists between different devices. As LibreTube doesn't use YouTube's official APIs, you neither require a YouTube account nor Google Services installed on your device.
```

Comment:
I think it would be better to first explain what LibreTube does (i.e. not use official APIs, be more private, better browsing, etc.) and after that mention Piped. Feels a bit weird to immediately start with an optional feature.
I would also avoid using technical terms like API, as these are pretty meaningless to most people. 
Maybe something more like this?

```
<i>LibreTube</i> is an alternative frontend for YouTube, focusing on user privacy, without requiring a YouTube account nor Google Services, and better watching experience. It optionally allows you to use <a href='https://github.com/TeamPiped/Piped' target='_blank' rel='nofollow'>Piped</a> to load data and play videos for better privacy as well as syncing your subscriptions and playlists between different devices.
```

Reply:
I think that goes into the right direction, but the first sentence feels a bit too long and complicated, so I think we should split it.

```
<i>LibreTube</i> is an alternative frontend for YouTube, focusing on user privacy and better watching experience. It doesn't require any YouTube account or Google Services installed on your device.

<br />

LibreTube optionally allows you to use <a href='https://github.com/TeamPiped/Piped' target='_blank' rel='nofollow'>Piped</a> to load data and play videos for better privacy as well as syncing your subscriptions and playlists between different devices.
```

Reply:
Sounds like a good idea.